extended-statistics.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 扩展统计信息简介
+aliases: ['/docs-cn/dev/extended-statistics/','/docs-cn/dev/reference/performance/extended-statistics/']
```

Comment:
It seems that this is a new file. For a newly added file, no need to add `aliases`. `aliases` is used to support previously used file names or links.

Reply:
Please add a `summary` line (a one-sentence brief intro) instead, which is good for SEO.